Classes/Controller/Backend/Order/OrderController.php

```diff
@@ -98,10 +109,10 @@ public function listAction(int $currentPage = 1): ResponseInterface
     public function exportAction(): ResponseInterface
     {
         $format = $this->request->getFormat();
+        $searchArguments = $this->request->getArgument('searchArguments');
+        $orderItems = $this->itemRepository->findAll();
```

Comment:
Why do remove the `$searchArguments` from the findAll action? The idea behind the filtering is, to only export a filtered list of orders. For example, only a specific time period can be exported.

Reply:
Sorry, I knew that this was your intention. I must have deleted it without noticing...